contrib/ansible/create-cluster-playbook.yaml

```diff
@@ -43,6 +43,9 @@
     # in the cert dir defined above.
     cluster_ca_name: ca
 
+  roles:
```

Comment:
can we just use 'import_role' https://docs.ansible.com/ansible/2.4/import_role_module.html in the 'tasks' section below instead of adding a 'roles' section? this should accomplish loading the contents/library of kubectl-ansible into the playbook so that the kubectl_apply and oc_process later will still work.